src/main/java/org/example/fuzzy_matcher/util/DataReader.java

```diff
@@ -0,0 +1,42 @@
+package org.example.fuzzy_matcher.util;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.example.objects.Music;
+import org.example.objects.MusicList;
+import org.example.objects.User;
+import org.example.objects.UserList;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+public class DataReader {
+
+    public static MusicList readMusicListFromJson(String filePath) throws Exception {
+        File jsonFile = new File(filePath);
+        ObjectMapper objectMapper = new ObjectMapper();
+        return objectMapper.readValue(jsonFile, MusicList.class);
+    }
+
+    public static UserList readUsersFromJson(String filePath) {
+        try {
+            File jsonFile = new File(filePath);
+            ObjectMapper objectMapper = new ObjectMapper();
+            return objectMapper.readValue(jsonFile, UserList.class);
+        } catch (IOException e) {
+            e.printStackTrace();
+            return null;
+        }
+    }
+
+    public static void main(String[] args) throws Exception {
```

Comment:
@SebasBarra02 This method shouldn't be here. Move it to the main we have or if you want to test your previous methods, you should write unit tests for it.